ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.tsx

```diff
@@ -332,3 +300,141 @@ export function FormRow(
   </>
 }
 
+
+function EditMailingAddressRow(
```

Comment:
Agreed. Thinking about it more, I think eventually this might be made more flexible and put in core so that address validation doesn't have to be remade everywhere in the app.